.github/workflows/run-e2e-tests.yaml

```diff
@@ -0,0 +1,78 @@
+name: run-azure-e2e-tests
+
+on:
+  workflow_call:
+    inputs:
+      #TODO how to decide/setup branch/versions etc
+      branch:
+        description: 'Branch to run tests on'
+        required: true
+        default: 'main'
+        type: string
+      bee-name:
+        description: 'bee name'
+        required: true
+        type: string
+      billing-project-name:
+        description: 'name of billing project'
+        required: true
+        type: string
+
+env:
+  CHECKOUT_PATH: "dsp-reusable-workflows/" # arbitrary folder name to checkout to for organizational purposes
+  SCRIPT_PATH: "e2e-test/" # location in the checked-out repo of the script to run
+  ARTIFACT_PATH: "test-results/" # arbitrary folder name to download test-results to for organizational purposes
+  BEE_NAME: '${{ inputs.bee-name }}'
+  BILLING_PROJECT_NAME: '${{ inputs.billing-project-name }}' # github token for access to kick off a job in the private repo
+  #AZURE_TOKEN: '${{ secrets.user-access-token }}' # github token for access to kick off a job in the private repo
+
+jobs:
+  init-github-context:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get inputs or use defaults
+        id: extract-inputs
+        run: |
+#          echo "owner-subject=${{ inputs.owner-subject              || 'hermione.owner@quality.firecloud.org' }}" >> "$GITHUB_OUTPUT"
```

Comment:
I think this debug output would be helpful and not too spammy to leave enabled.

Reply:
This is not debug but actually generating a variable for use in a later step, except that we don't need it in this case.  (Other actions that might need different/many users or to differentiate between owners and readers)